build/ci/production-values.yaml

```diff
@@ -85,51 +80,33 @@ varnish:
     between_bytes_timeout: 60  # Default 60s
   secrets:
     varnish:
-      ref: varnish-secret-production
+      ref: varnish-secret
 ingress:
-  hosts: 
-    - host: "sefaria.org"
-      path: /*
-      serviceName: nginx
-      port: 80
-    - host: "sefaria.org.il"
-      path: /*
-      serviceName: nginx
-      port: 80
-    - host: "www.sefaria.org"
-      path: /*
-      serviceName: nginx
-      port: 80
-    - host: "www.sefaria.org.il"
+  hosts:
+    - host: "contextus.org"
       path: /*
       serviceName: nginx
       port: 80
-    - host: "prod.sefaria.org"
+    - host: "www.contextus.org"
       path: /*
       serviceName: nginx
       port: 80
-    - host: "prod.sefaria.org.il"
+    - host: "prod.contextus.org"
       path: /*
       serviceName: nginx
       port: 80
-    - host: "rollout.sefaria.org"
-      path: /*
-      serviceName: nginx
-      port: 80
-    - host: "rollout.sefaria.org.il"
+    - host: "rollout.contextus.org"
       path: /*
       serviceName: nginx
       port: 80
   secrets:
-    originTls: 
-      ref: origin-tls-prod2
-    originIlTls:
-      ref: origin-il-tls-prod2 
+    originTls:
+      ref: origin-tls
```

Comment:
Did we pin down whether this should be origin-tls or origin-tls-prod2?  I think the sealed secret in infra is prod2?

Reply:
The secret is origin-tls in cloudflare-cert.yaml